text/0000-export-to-oci.md

```diff
@@ -0,0 +1,343 @@
+       ├── index.json
+       └── oci-layout
+```
+
+And the analyzer is invoked as follows
+
+```=shell
+> export CNB_USE_OCI=true
+> /cnb/lifecycle/analyzer -run-image cnbs/sample-stack-run:bionic my-app-image
+```
+
+Or
+
+```=shell
+> /cnb/lifecycle/analyzer -oci -run-image cnbs/sample-stack-run:bionic my-app-image
+```
+
+The expected output is the analysis metadata [analyzed.toml](https://github.com/buildpacks/spec/blob/main/platform.md#analyzedtoml-toml) with a content similar to this one:
+
+```=TOML
+[run-image]
+  reference = "sha256:eed6d69be53111ad1d7d3f5d1c037350e6807986feb479a67f36b15f9205a56d"
+```
+
+Where the [ImageID](https://github.com/opencontainers/image-spec/blob/main/config.md#imageid) is calculated according to the OCI specification
+
+
+### Exporting to OCI layout format
+
+Similar to the previous example, let's suppose the `<oci-dir>` or `CNB_OCI_PATH` environment variable points to a  directory that contains the `run-image` with the following format
+
+```=shell
+oci
+├── cnbs
+   └── sample-stack-run
+       ├── blobs
+       │   └── sha256
+       │       ├── 1f59171fcf9a8c2a78192d4dfbf88e6f0258e24f21798ffe015c07682d3a944a
+       │       ├── 219480c73c20efd82d425207e7555eba09cb113c845250cef09ba376e1dd506e
+       │       ├── 63882e64890d1adea5824dbb9bc6a812140d6b85752bf74a0a26fca78e1baf5a
+       │       ├── 63cb781c9d22ed765584437fea6db568f79be248ce40fc6695017d3ef3e8caaa
+       │       └── 6b534265b45a4c09a5e05dde5ccf2450ea505481658a1c9ae9e0aae32362e941
+       ├── index.json
+       └── oci-layout
+```
+
+And the exporter is invoked as follows
+
+```=shell
+> export CNB_OCI_PATH=true
+> /cnb/lifecycle/exporter -run-image cnbs/sample-stack-run:bionic my-app-image
+```
+
+Or
+
+```=shell
+>  /cnb/lifecycle/exporter -oci -run-image cnbs/sample-stack-run:bionic my-app-image
+```
+
+The expected output is the application `<image>` exported in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format
+
+```=shell
+oci
+├── cnbs
+  │   └── sample-stack-run
+  │       ├── blobs
+  │       │   └── sha256
+  │       │       ├── 1f59171fcf9a8c2a78192d4dfbf88e6f0258e24f21798ffe015c07682d3a944a
+  │       │       ├── 219480c73c20efd82d425207e7555eba09cb113c845250cef09ba376e1dd506e
+  │       │       ├── 63882e64890d1adea5824dbb9bc6a812140d6b85752bf74a0a26fca78e1baf5a
+  │       │       ├── 63cb781c9d22ed765584437fea6db568f79be248ce40fc6695017d3ef3e8caaa
+  │       │       └── 6b534265b45a4c09a5e05dde5ccf2450ea505481658a1c9ae9e0aae32362e941
+  │       ├── index.json
+  │       └── oci-layout
+  └── my-app-image
+      ├── blobs
+      │   └── sha256
+      │       ├── 14eaea7168b1fc4b8b30f7a20f7609335cc3dbcfb6d4c1afeb1e5daefd26cdf9
+      │       ├── 219480c73c20efd82d425207e7555eba09cb113c845250cef09ba376e1dd506e -> ../../../cnbs/sample-stack-run/blobs/sha256/219480c73c20efd82d425207e7555eba09cb113c845250cef09ba376e1dd506e
```

Comment:
No, the idea is the `lifecycle` should find all the required layers on disk in OCI and the platform must take care of that by downloading them. 

In your suggested scenario, the `exporter` should throw an error because the run-image will not be found on the disk, but I feel tempted to take the approach I described [here](https://github.com/buildpacks/rfcs/pull/203#discussion_r896144953), where the inputs for the phases are explicit with a `param-dir`